gcloud/monitoring/client.py

```diff
@@ -195,6 +203,130 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Construct a metric object.
+
+        The type of the Metric should match the type of a MetricDescriptor.
+        For a list of built-in Metric types and the associated labels, see:
+
+        https://cloud.google.com/monitoring/api/metrics
+
+        Here is an example of creating using a built-in metric type:
```

Comment:
I would recommend using a custom metric for this first example, and making it the only example in this docstring. I can't think of an everyday use-case for constructing a Metric instance for a built-in type.